### PR TITLE
Add support for releasing arm64 builds of charm

### PR DESCRIPTION
**Pull request by samuelallan72**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -13,17 +13,32 @@ jobs:
...
         uses: actions/checkout@v4
+
       - name: Initialize lxd  # This should dropped once it's implemented on charming-actions itself. https://github.com/canonical/charming-actions/issues/140
         uses: canonical/setup-lxd@v0.1.1
+
+        # NOTE: this action only supports a single charm file output https://github.com/canonical/charming-actions/blob/2.4.0/src/services/charmcraft/charmcraft.ts#L193-L194
+        # This is fine right now because the charm only has a single build per architecture,
+        # so only one will be built here.
       - name: Upload charm to charmhub
         uses: canonical/charming-actions/upload-charm@2.4.0
```
Comment (chanchiwai-ray) [inline .github/workflows/release.yaml:38]: it's there any concern bumping it to the latest (as of now: 2.6.2) ?

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -13,17 +13,31 @@ jobs:
...
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+
       - name: Initialize lxd  # This should dropped once it's implemented on charming-actions itself. https://github.com/canonical/charming-actions/issues/140
         uses: canonical/setup-lxd@v0.1.1
+
+        # NOTE: this action only supports a single charm file output https://github.com/canonical/charming-actions/blob/2.4.0/src/services/charmcraft/charmcraft.ts#L193-L194
+        # This is fine right now because the charm only has a single build per architecture,
+        # so only one will be built here.
```
Comment (chanchiwai-ray) [inline .github/workflows/release.yaml:36]: this should be added in 2.5.2rc

File: .github/workflows/release.yaml (hunk 1 above)
Comment (samuelallan72) [reply .github/workflows/release.yaml]: Updated

**Review by Pjack — APPROVED** [review]

**Review by gabrielcocenza — APPROVED** [review]

File: .github/workflows/release.yaml (hunk 2 above)
Comment (samuelallan72) [reply .github/workflows/release.yaml]: Thanks, good catch!  Updated. :)

**Review by chanchiwai-ray — APPROVED** [review]

**Comment by samuelallan72** [comment]
Now that release.yaml will be controlled from the automation repo ( https://github.com/canonical/charm-local-users/pull/23 ), and this has been approved locally, I'll close this and open a PR to the automation repo to add it.

**Comment by samuelallan72** [comment]
Actually, we probably could merge and test this first, then migrate it to the terraform repo later.  What do you think?

**Review by Pjack — APPROVED** [review]

**Pull request merged**